### PR TITLE
update readme for storage libraries to include BOM information.

### PR DESCRIPTION
**Pull request by pallavit**

### CODE REVIEW

**Comment by pallavit** [comment]
soft ping, review please.
/cc: @amishra-dev , @gapra-msft , @jaschrep-msft , @kasobol-msft , @rickle-msft

**Review by amishra-dev — CHANGES_REQUESTED** [review]
Putting a wait on this while the conversation on email is resolved.

File: sdk/storage/azure-storage-blob-batch/README.md [hunk 1]
```diff
@@ -17,6 +17,39 @@ definition, such as text or binary data.
 
 ### Include the package
 
+#### Include the BOM file
+
+Please include the azure-sdk-bom to your project to take dependency on GA version of the library. In the following snippet, replace the {bom_version_to_target} placeholder with the version number.
```
Comment (joshfree) [inline sdk/storage/azure-storage-blob-batch/README.md:22]: > Please include the azure-sdk-bom to your project to take dependency on GA version of the library.

Please include the azure-sdk-bom to your project to take **a** dependency on **the General Availability (GA)** version of the library.

File: sdk/storage/azure-storage-blob-cryptography/README.md [hunk 2]
```diff
@@ -18,6 +18,40 @@ This package supports client side encryption for blob storage.
 
 ### Include the package
 
+#### Include the BOM file
+
+Please include the azure-sdk-bom to your project to take dependency on GA version of the library. In the following snippet, replace the {bom_version_to_target} placeholder with the version number.
```
Comment (joshfree) [inline sdk/storage/azure-storage-blob-cryptography/README.md:23]: > Please include the azure-sdk-bom to your project to take dependency on GA version of the library.

Please include the azure-sdk-bom to your project to take **a** dependency on **the General Availability (GA)** version of the library.

File: sdk/storage/azure-storage-blob/README.md [hunk 3]
```diff
@@ -17,6 +17,39 @@ definition, such as text or binary data.
 
 ### Include the package
 
+#### Include the BOM file
+
+Please include the azure-sdk-bom to your project to take dependency on GA version of the library. In the following snippet, replace the {bom_version_to_target} placeholder with the version number.
```
Comment (joshfree) [inline sdk/storage/azure-storage-blob/README.md:22]: > Please include the azure-sdk-bom to your project to take dependency on GA version of the library.

Please include the azure-sdk-bom to your project to take **a** dependency on **the General Availability (GA)** version of the library.

File: sdk/storage/azure-storage-common/README.md [hunk 4]
```diff
@@ -13,6 +13,40 @@ Azure Storage Common is a library that has common dependencies for all Azure Sto
 
 ### Include the package
 
+#### Include the BOM file
+
+Please include the azure-sdk-bom to your project to take dependency on GA version of the library. In the following snippet, replace the {bom_version_to_target} placeholder with the version number.
```
Comment (joshfree) [inline sdk/storage/azure-storage-common/README.md:18]: > Please include the azure-sdk-bom to your project to take dependency on GA version of the library.

Please include the azure-sdk-bom to your project to take **a** dependency on **the General Availability (GA)** version of the library.

File: sdk/storage/azure-storage-file-datalake/README.md [hunk 5]
```diff
@@ -18,7 +18,38 @@ efficient data access.
 
 ### Include the package
 
-Add a dependency on Azure Storage File Datalake
+#### Include the BOM file
+
+Please include the azure-sdk-bom to your project to take dependency on GA version of the library. In the following snippet, replace the {bom_version_to_target} placeholder with the version number.
```
Comment (joshfree) [inline sdk/storage/azure-storage-file-datalake/README.md:23]: > Please include the azure-sdk-bom to your project to take dependency on GA version of the library.

Please include the azure-sdk-bom to your project to take **a** dependency on **the General Availability (GA)** version of the library.

File: sdk/storage/azure-storage-file-share/README.md [hunk 6]
```diff
@@ -20,6 +20,40 @@ Shares provide a way to organize sets of files and also can be mounted as an SMB
 
 ### Include the package
 
+#### Include the BOM file
+
+Please include the azure-sdk-bom to your project to take dependency on GA version of the library. In the following snippet, replace the {bom_version_to_target} placeholder with the version number.
```
Comment (joshfree) [inline sdk/storage/azure-storage-file-share/README.md:25]: > Please include the azure-sdk-bom to your project to take dependency on GA version of the library.

Please include the azure-sdk-bom to your project to take **a** dependency on **the General Availability (GA)** version of the library.

File: sdk/storage/azure-storage-queue/README.md [hunk 7]
```diff
@@ -15,6 +15,39 @@ A single queue message can be up to 64 KB in size, and a queue can contain milli
 
 ### Include the package
 
+#### Include the BOM file
+
+Please include the azure-sdk-bom to your project to take dependency on GA version of the library. In the following snippet, replace the {bom_version_to_target} placeholder with the version number.
```
Comment (joshfree) [inline sdk/storage/azure-storage-queue/README.md:20]: > Please include the azure-sdk-bom to your project to take dependency on GA version of the library.

Please include the azure-sdk-bom to your project to take **a** dependency on **the General Availability (GA)** version of the library.

**Review by joshfree — APPROVED** [review]

**Comment by joshfree** [comment]
Adding `Do Not Merge` label, so ensure only the Storage team merges this.

**Comment by pallavit** [comment]
Given our sync on Tuesday - can we merge these changes? 
/cc: @joshfree , @amishra-dev @kasobol-msft

**Review by rickle-msft — APPROVED** [review]

**Comment by pallavit** [comment]
@rickle-msft thanks for the review.

**Review by amishra-dev — APPROVED** [review]

**Comment by joshfree** [comment]
@rickle-msft @amishra-dev any concerns with merging this now?

**Pull request merged**